.ci/aws/Jenkinsfile

```diff
@@ -62,12 +62,21 @@ pipeline {
                     def p4d_region = "us-east-2"
                     def p4d_odcr = "cr-0e5eebb3c896f6af0"
                     def p4d_addl_args = "${addl_args_pr} --odcr ${p4d_odcr}"
+                    def p5_lock_label = "p5-1-4node"
+                    def p5_region = "us-east-2"
+                    def p5_odcr = "cr-0b5693c31bc89b82d"
+                    def p5_addl_args = "${addl_args_pr} --odcr ${p5_odcr}"
 
                     // p4d tests
                     stages["4_p4d_alinux2"] = common.get_test_stage_with_lock("4_p4d_alinux2", env.BUILD_TAG, "alinux2", "p4d.24xlarge", p4d_region, p4d_lock_label, num_instances, config, p4d_addl_args)
                     stages["4_p4d_ubuntu2004"] = common.get_test_stage_with_lock("4_p4d_ubuntu2004", env.BUILD_TAG, "ubuntu2004", "p4d.24xlarge", p4d_region, p4d_lock_label, num_instances, config, p4d_addl_args)
                     stages["4_p4d_ubuntu2204"] = common.get_test_stage_with_lock("4_p4d_ubuntu2204", env.BUILD_TAG, "ubuntu2204", "p4d.24xlarge", p4d_region, p4d_lock_label, num_instances, config, p4d_addl_args)
 
+                    // p5 tests
+                    stages["4_p5_alinux2"] = common.get_test_stage_with_lock("4_p5_alinux2", env.BUILD_TAG, "alinux2", "p5.48xlarge", p5_region, p5_lock_label, num_instances, config, p5_addl_args)
+                    stages["4_p5_ubuntu2004"] = common.get_test_stage_with_lock("4_5_ubuntu2004", env.BUILD_TAG, "ubuntu2004", "p5.48xlarge", p5_region, p5_lock_label, num_instances, config, p5_addl_args)
```

Comment:
```suggestion
                    stages["4_p5_ubuntu2004"] = common.get_test_stage_with_lock("4_p5_ubuntu2004", env.BUILD_TAG, "ubuntu2004", "p5.48xlarge", p5_region, p5_lock_label, num_instances, config, p5_addl_args)
```

Reply:
done, this is cosmetic b/c the cluster_name parameter is actually never used.

Reply:
this is used in pipeline overview, not blue ocean.